wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1489,13 +1518,15 @@ instance ToSchema NodeInfo where
       & ("syncProgress"          --^ "Syncing progression, in percentage.")
       & ("blockchainHeight"      --^ "If known, the current blockchain height, in number of blocks.")
       & ("localBlockchainHeight" --^ "Local blockchain height, in number of blocks.")
-      & ("localTimeDifference"   --^ "Local time difference, in number of blocks.")
+      & ("localTimeInformation"  --^ "Information about the clock on this node.")
```

Comment:
Whoops, sorry for that, looks like a wrong copy/paste from me :s ... 
Good catch :+1: